answerking_app/tests/API_types.py

```diff
@@ -0,0 +1,26 @@
+from typing import TypedDict
+
+
+class IDType(TypedDict):
+    id: int
+
+
+class NewItemType(TypedDict):
+    name: str
+    price: str
+    description: str
+    stock: int
+    calories: int
+
+
+class ItemType(IDType, NewItemType):
+    pass
+
+
+class NewCategoryType(TypedDict):
+    name: str
+    items: list[ItemType]
+
+
+class CategoryType(IDType, NewCategoryType):
+    pass
```

Comment:
This is something we should probably be discussing as a group. Have you reached out to the other guys to see if they are implementing their tests in the same way as this?

Reply:
Cool as long as everyone who starts a testing refactor ticket is on board